charts/incubator/automatic-ripping-machine/values.yaml

```diff
@@ -0,0 +1,42 @@
+image:
+  repository: tccr.io/truecharts/automatic-ripping-machine
+  pullPolicy: IfNotPresent
+  tag: v2.6.60@sha256:80794ea2bcf0372856f83c980ee65f26eacd3253b9c97215e686fb85e2df969b
+
+securityContext:
+  container:
+    runAsNonRoot: false
+    readOnlyRootFilesystem: false
+    runAsUser: 0
+    runAsGroup: 0
+    allowPrivilegeEscalation: true
+
+service:
+  main:
+    ports:
+      main:
+        protocol: http
+        targetPort: 8080
+        port: 10683
+
+workload:
+  main:
+    podSpec:
+      containers:
+        main:
+          env:
+            # default media perms
+            ARM_UID: 568
+            ARM_GID: 568
```

Comment:
Link them to `{{ .Values.securityContext.container.PUID }}` and `{{ .Values.securityContext.pod.fsGroup }}`